test/jake/jakelib/migrate-test.js

```diff
@@ -543,6 +543,100 @@ test('migrate:down undos only the last run migration', (temp) => {
     });
 });
 
+test('list prints migrations both completed and pending', async (temp) => {
+  const migrationFile1 = '001_create_animals_table.js';
+  const migrationFile2 = '002_add_age_column_to_animals_table.js';
+
+  let { stdout } = await assertExec(
```

Comment:
does this need to be `let`? I assume we don't overwrite it anywhere.

Reply:
It is getting used below as well.

Reply:
Can we just have to separate instances of stdout then? Reassigning variables in this fashion is confusing and is better avoided.